### PR TITLE
adds syntax-scoped properties example to README.md

### PR DESCRIPTION
**Pull request by mjrk**
See #49 and #51

### CODE REVIEW

**Comment by thedaniel** [comment]
Is jade the only language that disables this by default?

**Comment by lee-dohm** [comment]
Markdown is also disabled, but by a special case in the code here:

https://github.com/atom/whitespace/blob/master/lib/whitespace.coffee#L59-L63

**Comment by nathansobo** [comment]
This looks good to me. @lee-dohm can you see any reason not to merge?

**Comment by lee-dohm** [comment]
No, I don't. There's a spelling error that I just caught on a final look, but that can be fixed after it is merged too.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
 Strips trailing whitespace and adds a trailing newline when an editor is saved.
+
+To disable/enable features for a certain language package, you can use 
+syntax-scoped properties in your `config.cson`. E.g.
+
+    '.slim.text':
+      whitespace: 
+        removeTrailingWhitespace: false
+
+You find the `scope` on top of a grammer package's settings view.
```
Comment (lee-dohm) [inline README.md]: `grammer` &rarr; `grammar`

**Comment by lee-dohm** [comment]
Going ahead and merging this and fixing the typo myself. Thanks @mjrk!

**Pull request merged**